src/components/dataset/list.vue

```diff
@@ -10,10 +10,17 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <div>
-    <dataset-introduction/>
-    <dataset-table/>
-    <loading :state="datasets.initiallyLoading"/>
+  <div id="dataset-list-page">
```

Comment:
Would you mind making it just `dataset-list`? I've been trying to keep the HTML id and the component name in sync.